scalanet/test/src/io/iohk/scalanet/peergroup/UDPPeerGroupSpec.scala

```diff
@@ -12,6 +12,7 @@ import io.iohk.scalanet.peergroup.ControlEvent.InitializationError
 import org.scalatest.concurrent.ScalaFutures._
 import io.iohk.scalanet.peergroup.PeerGroup.{ChannelAlreadyClosedException, MessageMTUException}
 import io.iohk.scalanet.peergroup.StandardTestPack._
+import io.iohk.scalanet.peergroup.udp.DynamicUDPPeerGroup
```

Comment:
We probably should rename whole spec to `DynamicUDPPeerGroupSpec`

Reply:
I wasn't sure yet if we can reuse this spec to test both implementations. Seems to delegate to some "standard tests" so I thought this might be the case. If not I'll rename it.

Reply:
that sounds reasonable, let's leave it as it is then.